config/locales/en/settings.yml

```diff
@@ -240,3 +240,13 @@ en:
       more_info_link_description: Link shown in the consent banner to the cookie policy page.
       version_name: Version name
       version_name_description: Change the version name to force all users to go through the cookie consent process again. This feature can be handy when introducing drastic cookie policy changes requiring a new user acknowledgement.
+    llm:
+      title: LLM Settings
+      provider: "LLM Provider"
+      provider_description: "Providers will be disabled until credentials are configured in the secrets.yml."
+      provider_blank_option: "None"
+      model_blank_option: "None"
```

Comment:
These two translations could probably be removed, as they're no longer referenced in the code.